packages/base-map/src/index.tsx

```diff
@@ -143,6 +150,24 @@ const BaseMap = ({
           <ul
             className={`layers-list ${fakeMobileHover && "fake-mobile-hover"}`}
           >
+            {typeof baseLayer === "object" &&
+              baseLayer.map((layer: string, index: number) => {
+                return (
+                  <li key={index}>
```

Comment:
We can probably use `layer` as the key instead of the index.

Reply:
I had this but key `0` caused problems